graphql/connection_test.go

```diff
@@ -280,7 +280,7 @@ func TestConnection(t *testing.T) {
 
 	snap.SnapshotQuery("Pagination, filter", `{
 		inner {
-			filterByCan: innerConnectionWithFilter(filterText: "can", first: 5, after: "") {
+			filterByCan: innerConnectionWithFilter(filterText: "can", filterTextOption:["foo","wug"],first: 5, after: "") {
```

Comment:
how about `filterTextFields` to match the `TextFilterFields` naming we use to declare the text filter fields